### ISSUE TITLE
Show replies from users we follow and users they follow

### ISSUE THREAD

**Issue by shanecb** [planetary-social/nos#106]
Currently the RepliesView only shows posts from users the current user follows. The RepliesView should show replies from users the current user follows **and** replies from any users they follow ~~the original poster of the root message follows.~~. So if each user's social graph is a tree where each node is a user and each line represents a follow, we want to show replies from the first 3 layers (The root node plus the two layers below it) of the graph.

**Comment by mplorentz** [comment]
@shanecb I tried to edit this to be more clear, let me know if I failed haha.

This is an algorithm we are bringing over from scuttlebutt. The thinking goes that you probably mostly follow people you respect and share some basic values with. If they do the same then there shouldn't be many spammers or abusive folks in this "2 hops" network. 

I'd be interested to hear other algorithms that accomplish the same goal though if you have ideas.
